### PR TITLE
Fix/improve logic for convective transportable tracers in CCPP, add GFS_checktracer debugging routine, correct spelling "janic" --> "janjic"

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

- Update submodule pointer for the changes described in https://github.com/NCAR/ccpp-physics/pull/752 (fix/improve logic for convective transportable tracers in CCPP, add GFS_checktracer debugging routine)
- Add missing `active` attribute for one variable in `GFS_typedefs.meta` and remove a legacy comment line in `GFS_typedefs.F90`
- Correct spelling "janic" --> "janjic" in a few CCPP standard names (thanks @DusanJovic-NOAA)

### Issue(s) addressed

- fixes https://github.com/NCAR/ccpp-physics/issues/751

## Testing

For regression testing, see https://github.com/ufs-community/ufs-weather-model/pull/875

## Dependencies

https://github.com/NCAR/ccpp-physics/pull/752
https://github.com/NOAA-EMC/fv3atm/pull/413
https://github.com/ufs-community/ufs-weather-model/pull/875

### CODE REVIEW

File: ccpp/data/GFS_typedefs.meta [hunk 1]
```diff
@@ -613,6 +613,7 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+  active = (flag_for_mellor_yamada_janic_surface_layer_scheme .or. flag_for_mellor_yamada_janic_pbl_scheme .or. (control_for_land_surface_scheme == identifier_for_noah_wrfv4_land_surface_scheme))
```
Comment (DusanJovic-NOAA) [inline ccpp/data/GFS_typedefs.meta]: 'janic' should be spelled as 'janjic'

Comment (climbfuji) [reply ccpp/data/GFS_typedefs.meta]: Thanks for catching this. This turns out to be wrong in a few places. Will make the PR bigger but won't change the results.

Comment (climbfuji) [reply ccpp/data/GFS_typedefs.meta]: Fixed, please have a look.

Comment (DusanJovic-NOAA) [reply ccpp/data/GFS_typedefs.meta]: Thanks for fixing this.

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]

**Comment by climbfuji** [comment]
Superseded by https://github.com/NOAA-EMC/fv3atm/pull/416.

**Pull request closed without merging**